src/index.ts

```diff
@@ -4,6 +4,23 @@ function isWildcard(accept: string | null) {
   return accept && !!accept.split(',').find(x => x.match(/^\s*\*\/\*/))
 }
 
+// We don't want to add `@types/trusted-types` as a dependency, so we use this stand-in.
+
+interface CSPTrustedHTMLToStringable {
+  toString: () => string
+}
+interface CSPTrustedTypesPolicy {
+  createHTML: (s: string, response: Response) => CSPTrustedHTMLToStringable
+}
+let cspTrustedTypesPolicyPromise: Promise<CSPTrustedTypesPolicy> | null = null
+// Passing `null` clears the policy.
+export function setCSPTrustedTypesPolicy(policy: CSPTrustedTypesPolicy | Promise<CSPTrustedTypesPolicy> | null): void {
+  cspTrustedTypesPolicyPromise = policy === null ? policy : Promise.resolve(policy)
+}
+
+// TODO: find another way to make this available for testing.
+;(globalThis as any).setCSPTrustedTypesPolicy = setCSPTrustedTypesPolicy
```

Comment:
The test doesn't actually have a way to use exported functions. Do you have a recommendation for how to get access to `setCSPTrustedTypesPolicy` from `test.js` without this hack?

Reply:
Apologies I thought we had updated to `web-test-runner`. I've now done so in https://github.com/github/include-fragment-element/pull/86 so if you rebase it should be evident how to import the function now, as there is an `import` statement at the top of the file.